Elearning.txt

```diff
@@ -0,0 +1,77 @@
+
+create table Student(StudentId int primary key identity(1,1),
+Name varchar(50) not null,
+PhoneNumber nvarchar(15),
+Email varchar(254) unique);
+
+create table Instructor(InstructorId int primary key identity(1,1),
+Name varchar(50) not null,
+PhoneNumber nvarchar(15),
+Email varchar(254) unique);
+
+create table Courses(CourseId int primary key identity(1,1),
+Title varchar(50) not null,
+Description Text,
+NoOfLessons int,
+InstructorId int not null,
+foreign key(InstructorId) references Instructor(InstructorId),
+);
+
+create table Lessons(LessonId int primary key identity(1,1),
+Title varchar(50),
+Description Text,
+CourseId int,
+foreign key(CourseId) references Courses(CourseId));
+
+create table Review(ReviewId int primary key identity(1,1),
+ReviewType varchar(50));

```

Comment:
what is the use of this table ?

Reply:
(e.g., peer-reviewed, auto-graded, instructor-graded),type of assignment